app/components/elements/link/index.js

```diff
@@ -1,21 +1,11 @@
-import check from 'check-arg-types'
-import {pathEq} from 'wasmuth'
-
-import {getState} from '/store'
-
-import {click} from './actions'
-
-const toType = check.prototype.toType
-const isActive = (to) => pathEq(['url'], to, getState())
-
-export const Link = ({to, activeClass, className, children, ...props}) =>
-  <a
-    onclick={(e) => toType(to) === 'function' ? to(e) : click(to)(e)}
-    href={toType(to) === 'string' ? to : '#'}
-    className={`${props.class || className} ${activeClass && isActive(to) ? activeClass : ''}`}
-    {...props}
-  >
-    {children}
-  </a>
-
-export default Link
+import {route} from 'preact-router'
```

Comment:
Again, feels like overkill to switch files for such a tiny component. This also couples us to `preact-router` which we have already identified potential issues with. I rather keep the components uncoupled from the router.

Reply:
I don't think this couples us to preact-router because if we switch how we do routing all we need to change is the Link component. Since it will always be a SPA we'll always need to hijack link clicks and do something.

Reply:
Just to be clear about how routing works righ tnow. To change route after, for example, form success, you would call preact-router's `route` and that would be it. The state will get updated. And the link prevents default and calls `route` because preact-router listening to clicks on elements with href tags caused that weird bug before when new components get rendered. But if we switch to any other routing mechanism, we just need to replace calls to preact-router `route`